### DISCUSSION TITLE
ECOERA Millenium Biochar (Puro.earth & European Biochar Certificate)

### DISCUSSION THREAD

**Discussion by hai-yr** [opentaps/open-climate-investing#36] [Offsets]
https://puro.earth/CORC-co2-removal-certificate/biochar-ecoera-millennium-1-sweden-100015
https://puro.earth/CORC-co2-removal-certificate/biochar-ecoera-millennium-1-sweden-100015?print_view=1&popup=1&download=1

"The Biochar from the Hammenhög facility corresponding to ECOERA Millennium 1 is used is three main storage systems:
1: Agricultural fields, close to the production facility.
2: Green Urban Infrastructure - green roofs, tree plantations, and rain gardens. Reference: www.biokol.org
3: Soccer fields where the biochar is part of the bedding material preventing water and nutrient runoff as well as increasing the density of the grass-root zone and hence the total biomass. Reference: Lund Municipality.

The LCA system boundary as defined in Puro.earth standard covers cradle-to-gate GHG impact. Outside of LCA our biochar has the following climate impacts:
1: Biochar has agricultural benefits, and our own growth trials have resulted in 10-33% yield increase.
2: Use of biochar can lower the need for artificial fertilizers. Most of our biochar is applied to farmlands and avoid N2O emissions from fertilizers.
3: Methane emission reduction through avoided decomposing of feedstock biomass to CO2 and CH4. Our biochar is made of biomass residues from grain production which would otherwise be composted or burnt. Those emissions are avoided by stabilizing the GHG into our biochar.
4: Replacement of synthetic turf grass in soccer fields. Synthetic turf is made from polyethylene, which is a form of plastic.

Avoided decomposing of biomass: To make biochar we use agricultural biomass residue from grain production, which would in normal case be composted or burnt. Those emissions are avoided by stabilizing the carbon in the residue into our biochar for hundreds of years. 230 kg of CO2eq per mt ton biomass residue is avoided.
Avoided emissions from fertilizers and irrigation: If the biochar had not been spread to farms, parks, and soccer fields, they would have needed more fertilizers and water to grow and remain green. Making mineral fertilizers would have caused emissions and also spreading them and irrigating.

The payments for the carbon removal would increase their capacity to produce biochar in three ways:
1: a project start for installing an optimized version of the current system, with the potential to increase the size of the pyrolysis unit for lowering of CAPEX, increasing carbon sequestration.
2: usage of algae biomass for biochar production
3: usage of urban carbon streams (biosolids) as feedstock"

**Comment by hai-yr** [comment]
From what I can tell, their claim to additionality is that selling removals gives them more money to increase biochar production and carbon removal... however, it seems challenging to quantify or prove without detailed financials, no?

**Reply by sichen1234** [reply]
That's almost always true but is not strong enough of a claim to be an offset IMO.  

For example, buying renewable energy certificates (RECs) gives renewable energy developers, which should encourage more renewable energy development.  But Salesforce does not agree:
"In states with strong renewable energy targets, demand for unbundled RECs is often high and consistent. Voluntary purchasing of RECs in these markets drives demand (and price) even higher, encouraging the development of new renewable energy. Unfortunately, the demand for unbundled RECs is low in most markets, especially those without state-mandated targets. In such markets, unbundled RECs are not a significant source of revenue for renewable energy projects, and therefore do little to drive additional renewable energy development."

(See [Salesforce Clean Energy Strategy](https://pdf4pro.com/amp/view/salesforce-s-clean-energy-strategy-4efb7b.html))

**Comment by hai-yr** [comment]
My most recent thought on biochar as well as offset projects is: verification standards are necessary but not sufficient. Projects need to be rated (e.g. AAA to D) by independent parties.

**Comment by sichen1234** [comment]
I found out that biochar is in fact a product that is sold ([GivingGreen](https://www.givinggreen.earth/carbon-offsets-research/biochar#:~:text=Biochar%20is%20a%20product%20that,it%20is%20added%20to%20soils))

So to be an offset, we really need to prove that biochar was correctly used (real), that it wouldn't have been used without the offset (additional), and it didn't cause some other behavior to negate the project (leakage.)

For example, if the biochar would've been bought and used any way, then it's not additional.  Some places actually require biochar by law.  The biochar methodologies all check for this.  Are there other places where biochar is normally bought and used?

If people got free or deeply discounted biochar and then sold it, or threw them out, then it's not real.

Finally if by giving out free or deeply discounted biochar, we're just displacing local biochar businesses, then that's serious leakage.

**Comment by hai-yr** [comment]
<img width="668" alt="Screen Shot 2022-09-06 at 7 05 22 AM" src="https://user-images.githubusercontent.com/39865976/188656141-bb769921-30d2-4665-afee-561a82ff26b8.png">
<img width="677" alt="Screen Shot 2022-09-06 at 7 05 12 AM" src="https://user-images.githubusercontent.com/39865976/188656135-9d14ec6f-a601-4143-af49-cd287efad713.png">
